Godeps/Godeps.json

```diff
@@ -2926,6 +2943,66 @@
 			"Rev": "a69f0f19d246419bb931b0ac8f4f8d3f3e6d4feb"
 		},
 		{
+			"ImportPath": "google.golang.org/appengine",
```

Comment:
We have those vendored already, just not in Godeps.json. Since I've fixed godeps in 1.5 only Alexey picked those in 1.4. It's ok.